homeassistant/components/salt/device_tracker.py

```diff
@@ -0,0 +1,83 @@
+"""Support for Salt Fiber Box routers."""
+import logging
+
+from saltbox import SaltBox
+import voluptuous as vol
+
+from homeassistant.components.device_tracker import (
+    DOMAIN,
+    PLATFORM_SCHEMA,
+    DeviceScanner,
+)
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_IP = "192.168.1.1"
+DEFAULT_USERNAME = "admin"
```

Comment:
Don't add any such defaults. This would push bad use of standard accounts. Just add a note in docs that it's common.

Reply:
I saw this done in other router device trackers so I included the default arguments, but I suppose it's easy for the user to add these to the configuration.